server/portal/apps/message/views.py

```diff
@@ -1,12 +1,13 @@
 """
-.. :module: apps.intromessages.views
+.. :module: apps.message.views
```

Comment:
@rstijerina I've tried making it `messages` initially. But apparently there is a conflict with the built-in django `messages` utility if I make the app name that. So, it was throwing an error. So I opted to use `message`. I do think it would make more sense if the app name was plural. But I couldn't think of a generic name that could replace it.

Reply:
Ahh. What about announcements or portal_messages?

Reply:
Yeah! let's keep parity with the app